server/controllers/shared/caseListController.ts

```diff
@@ -0,0 +1,29 @@
+import type { Request, Response, TypedRequestHandler } from 'express'
+
+import type { ReferralService } from '../../services'
+import { ReferralUtils, TypeUtils } from '../../utils'
+import type { Caseload } from '@prison-api'
+
+export default class CaseListController {
+  constructor(private readonly referralService: ReferralService) {}
+
+  show(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      TypeUtils.assertHasUser(req)
+
+      const { caseloads, username } = res.locals.user
+
+      const activeCaseLoad = caseloads.find((caseload: Caseload) => caseload.currentlyActive)
+      const activeCaseLoadId = activeCaseLoad.caseLoadId
```

Comment:
annoying comment, but does this need to be assigned to a variable if it's just being used in the next line and nowhere else, and it's not doing anything too complicated that would be good to encapsulate under a helpful name?